torch/quantization/observer.py

```diff
@@ -938,7 +952,7 @@ def _combine_histograms(self, orig_hist, new_hist, upsample_rate, downsample_rat
         return orig_hist
 
     def forward(self, x_orig):
-        # type: (Tensor) -> Tensor
+        # type: (torch.Tensor) -> torch.Tensor
```

Comment:
Why not move this annotation inline, i.e. `def forward(self, x_orig: torch.Tensor) -> torch.Tensor:`?

Reply:
for some reason moving the annotation inline causes jit override issue
```
  File "/data/users/rongr/pytorch/torch/jit/_recursive.py", line 304, in create_methods_and_properties_from_stubs
    concrete_type._create_methods_and_properties(property_defs, property_rcbs, method_defs, method_rcbs, method_defaults)
RuntimeError: Can't redefine method: forward on class: __torch__.torch.quantization.observer.HistogramObserver
```